src/main/interface/settings.c

```diff
@@ -842,7 +842,7 @@ const clivalue_t valueTable[] = {
     { "smart_feedforward",          VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_PID_PROFILE, offsetof(pidProfile_t, smart_feedforward) },
 #endif
 #if defined(USE_ITERM_RELAX)
-    { "iterm_relax",                VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ITERM_RELAX }, PG_PID_PROFILE, offsetof(pidProfile_t, iterm_relax) },
+    { "iterm_relax_axis",           VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ITERM_RELAX }, PG_PID_PROFILE, offsetof(pidProfile_t, iterm_relax_axis) },
```

Comment:
cli setting should not get rename when there isnt any functional change

Reply:
It now matches to what it does and seems neater to be the same like the config params

Reply:
just because you also renamed it ;)
cli setting got this name for a reason and it should stay. it already is used in plenty of other places.

Reply:
@borisbstyle - There is no benefit to the users who already customised this setting, changing the name will only annoy them.  Technically this setting is not just axis, it also changes directionality.  If this setting was only axis choice, I would agree with you, but it isn't.  I do agree that the options are not self-evident, but changing the name doesn't improve that. Some of these settings may disappear with further testing, perhaps that we could re-think the name if we later change the options or the functionalities.
I would be really grateful that you don't change this CLI name at this point.

Reply:
@borisbstyle if you read @ctzsnooze you would notice that he exactly wants the same as this strange @wind0r

he wrote that a internal name change to iterm_relax_axis is fine but the external name should stay iterm_relax

Reply:
If you read my comment, where I say I don't mind and check the change than you will see that your advice is followed.

Reply:
i read it but it did not answered ctznooze question. it just blamed some wind0r for wanting to have it inconsistent. even when he and other also want it that way.

Reply:
@wind0r Blame is not correct word IMO. I was saying that the confusion may be expected with inconsistencies.

Reply:
@borisbstyle I just took that message to personal. Sorry for my mean response.

We just prefer to maybe be a little confused about that different name in cli instead of renaming it for every user. With in the last year's we got way many issues/requests about such small renames. So we want to avoid them as good as possible.

Reply:
I agree, and I am confident we all agree, that naming should, ideally, be consistent, right through the code.  But 'perfection' in this regard comes at a cost, particularly if we are talking about changing the CLI name.  In this case, I'm sure we all agree, the 'gain' of being utterly consistent isn't really worth the 'pain'.  The internal name is easy to discover from the external name, hence on this occasion isn't really a problem.  If the internal and external names were totally different that would be I think make a change of the external name worthwhile, perhaps.  Anyhow, this is now resolved - and I'm glad that we clarified our respective positions.